symfony/mailer/4.3/manifest.json

```diff
@@ -9,7 +9,7 @@
         "docker-compose.override.yml": {
             "services": [
                 "mailer:",
-                "  image: schickling/mailcatcher",
+                "  image: sj26/mailcatcher",
```

Comment:
This is the new official image see discussion in https://github.com/symfony/recipes/pull/1279. Thx @ker0x